scripts/asmCheck.py

```diff
@@ -7,36 +6,36 @@
 args = parser.parse_args()
 
 try:
-  sst_out = open(args.sstOutfile, 'r')
-except:
-  print("Cannot open file " + args.sstOutfile)
-  exit(1)
+    sst_out = open(args.sstOutfile, 'r')
+except Exception:
```

Comment:
`except:` catches _all_ exceptions, even `CTRL-C` on the keyboard. By specifying the `Exception` base class, we ensure that this is a true Python exception and not an external signal. `flake8` reports an error if bare `except` is used.